[2023-01-17 22:49:54,880] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: testing_dags_20.transition_best_model_version_to_prod_task 2023-01-17T22:40:48.062990+00:00 [queued]>
[2023-01-17 22:49:54,926] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: testing_dags_20.transition_best_model_version_to_prod_task 2023-01-17T22:40:48.062990+00:00 [queued]>
[2023-01-17 22:49:54,928] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2023-01-17 22:49:54,929] {taskinstance.py:1043} INFO - Starting attempt 1 of 2
[2023-01-17 22:49:54,930] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2023-01-17 22:49:54,959] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): transition_best_model_version_to_prod_task> on 2023-01-17T22:40:48.062990+00:00
[2023-01-17 22:49:54,980] {standard_task_runner.py:52} INFO - Started process 1597 to run task
[2023-01-17 22:49:54,996] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'testing_dags_20', 'transition_best_model_version_to_prod_task', '2023-01-17T22:40:48.062990+00:00', '--job-id', '15', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/modeling_dag.py', '--cfg-path', '/tmp/tmpjzpw40_a', '--error-file', '/tmp/tmp4ftsj0fp']
[2023-01-17 22:49:55,011] {standard_task_runner.py:77} INFO - Job 15: Subtask transition_best_model_version_to_prod_task
[2023-01-17 22:49:55,242] {logging_mixin.py:104} INFO - Running <TaskInstance: testing_dags_20.transition_best_model_version_to_prod_task 2023-01-17T22:40:48.062990+00:00 [running]> on host bb9ce9751eb1
[2023-01-17 22:49:55,422] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=email@example.com
AIRFLOW_CTX_DAG_OWNER=me
AIRFLOW_CTX_DAG_ID=testing_dags_20
AIRFLOW_CTX_TASK_ID=transition_best_model_version_to_prod_task
AIRFLOW_CTX_EXECUTION_DATE=2023-01-17T22:40:48.062990+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-01-17T22:40:48.062990+00:00
[2023-01-17 22:49:56,371] {logging_mixin.py:104} WARNING - 2023/01/17 22:49:56 WARNING mlflow.tracking.context.git_context: Failed to import Git (the Git executable is probably not on your PATH), so Git SHA is not available. Error: Failed to initialize: Bad git executable.
The git executable must be specified in one of the following ways:
    - be included in your $PATH
    - be set via $GIT_PYTHON_GIT_EXECUTABLE
    - explicitly set via git.refresh()

All git commands will error until this is rectified.

This initial warning can be silenced or aggravated in the future by setting the
$GIT_PYTHON_REFRESH environment variable. Use one of the following values:
    - quiet|q|silence|s|none|n|0: for no warning or exception
    - warn|w|warning|1: for a printed warning
    - error|e|raise|r|2: for a raised exception

Example:
    export GIT_PYTHON_REFRESH=quiet
[2023-01-17 22:49:56,373] {logging_mixin.py:104} WARNING - 
[2023-01-17 22:49:57,138] {python.py:118} INFO - Done. Returned value was: None
[2023-01-17 22:49:57,182] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=testing_dags_20, task_id=transition_best_model_version_to_prod_task, execution_date=20230117T224048, start_date=20230117T224954, end_date=20230117T224957
[2023-01-17 22:49:57,262] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-01-17 22:49:57,305] {local_task_job.py:146} INFO - Task exited with return code 0
